Source/DafnyDriver/BoogieAffectingOptions/NonZeroExitCodeIfErrorsOption.cs

```diff
@@ -0,0 +1,18 @@
+namespace Microsoft.Dafny;
+
+public class NonZeroExitCodeIfErrorsOption : BooleanOption {
+  public static readonly NonZeroExitCodeIfErrorsOption Instance = new();
+  public override object DefaultValue => true;
+  public override string LongName => "nonzero-exit-code-if-errors";
+  public override string ShortName => null;
+  public override string ArgumentName => "boolean";
+
+  public override string Description => @"
+if false then always exit with a 0 exit code, regardless of whether errors are found.
+If true (default) then use the appropriate exit code.".TrimStart();
```

Comment:
It's a bit uncommon to have such a flag.  Shouldn't we find another solution in the test suite itself?

Reply:
We already have this option in a substantial portion of our 1400+ tests, whenever you have %dafny_0, it means you have this flag.
Since `%baredafny verify` only takes new arguments, we need to expose this option to make it work.
I don't know of another simple way.

What we could do is hide this option to users, but I don't know how to do that. @keyboardDrummer ?

Reply:
But the description of that flag says "deprecated" in the old CLI:

```
  /countVerificationErrors:<n>
      (deprecated)
      0 - Set exit code to 0 regardless of the presence of any other errors.
      1 (default) - Emit usual exit code (cf. beginning of the help message).
```

Reply:
I think for this test we should just use the old CLI.  And we can then (in a separate PR) enhance LitTests.cs to accept `|| true` at the end of a command line, which is the "right" way to ignore an exit code.  WDYT?